### ISSUE TITLE
Terminal tab/name clears out (empty)

### ISSUE THREAD

**Issue by AmirSavand** [Microsoft/vscode#38786]
- VSCode Version: 1.18.1
- OS Version: Windows 10

Steps to Reproduce:

1. Open terminal
2. Type in `exit` then open another terminal immediately (with shortcut keys)

You will see the tab is empty, `exit` again it will become ok.

Reproduces without extensions: Yes

![untitled](https://user-images.githubusercontent.com/4629328/33021874-bd2da208-ce1c-11e7-8933-cf1131abc9e0.png)

**Comment by warpdesign** [comment]
I guess I found a bug that's related or the same as this one.

1. Open the integrated terminal
2. Open a new tab by cliking on `+`
3. Click on the terminal selection widget

It's empty.

Closing it, and opening it again fixes the problem.

Note that this happens after a brand new start of VSCode, I'm not sure it can be reproduced after that.

![vscode_terminal_tabs](https://user-images.githubusercontent.com/199648/33063867-1bd972b4-cea4-11e7-9742-9b1f1972f031.gif)
